src/Access/Common/AccessType.h

```diff
@@ -89,7 +89,7 @@ enum class AccessType
     M(CREATE_TEMPORARY_TABLE, "", GLOBAL, CREATE) /* allows to create and manipulate temporary tables;
                                                      implicitly enabled by the grant CREATE_TABLE on any table */ \
     M(CREATE_FUNCTION, "", GLOBAL, CREATE) /* allows to execute CREATE FUNCTION */ \
-    M(CREATE_NAMED_COLLECTION, "", GLOBAL, CREATE) /* allows to execute CREATE NAMED COLLECTION */ \
```

Comment:
This is not related to this PR but still.
This line means `GRANT CREATE` will grant also `GRANT create named collection`.
That doesn't seem good because `GRANT CREATE` was initially only about tables. Probably we shouldn't grant `create named collection` implicitly.

What do you think about changing that to
```
M(CREATE_NAMED_COLLECTION, "", NAMED_COLLECTION, NAMED_COLLECTION)
M(NAMED_COLLECTION, "", NAMED_COLLECTION, ALL)
```
?

Reply:
done